### DISCUSSION TITLE
怎么不能检查更新

### DISCUSSION THREAD

**Discussion by Glimspe** [shuax/MouseInc.Settings#389] [Q&A]
每次检查更新的时候都会提示“检查更新失败，请稍后再试”，

**Comment by Glimspe** [comment]
“设置通道连接失败,可能是Mouselnc主程序未更新”这种情况下只能重新下载软件更新吗？

**Comment by shuax** [comment]
https://github.com/shuax/MouseInc.Settings/releases/tag/2022%E5%B9%B44%E6%9C%889%E6%97%A5
你实在不更新只有找当时的设置界面
